metrics/trec_eval/trec_eval.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
```

Comment:
2022? not that it really matters :joy:

Reply:
I will do a separate PR to do this globally :)